src/main/java/com/patchworkmc/mapping/remapper/ManifestRemapper.java

```diff
@@ -23,17 +53,30 @@ public String remapMemberDescription(String descriptor) {
 
 	@Override
 	public String remapFieldName(String owner, String name, String descriptor) {
```

Comment:
From here down is very inconsistent on what format we accept and what we return (specifically dot or slash). Need to make it do one thing and then doc it.

Reply:
I think the AT parser should rewrite the classes to use `/` because that is consistent with Java internal names. I can fix this if needed. It's just an inconsistency because ATs use `.` instead.

Reply:
Some of the methods return / and some of them return ., so this needs to be fixed